packages/amplify/amplify_flutter/lib/amplify_flutter.dart

```diff
@@ -21,6 +21,9 @@ import 'src/amplify_impl.dart';
 
 export 'package:amplify_core/amplify_core.dart' hide Amplify;
 
+// ignore: depend_on_referenced_packages
```

Comment:
Since you added it to the pubspec, this is no longer needed. In general, `ignore` is almost never the right option. The lint prevents publishing errors.

Reply:
I thought this was the case as well - but the linter is still complaining.